azure/container.go

```diff
@@ -50,18 +50,25 @@ func (c *container) Item(id string) (stow.Item, error) {
 	return item, nil
 }
 
-func (c *container) Items(prefix, cursor string, count int) ([]stow.Item, string, error) {
+func (c *container) Browse(prefix, delimiter, cursor string, count int) ([]string, []stow.Item, string, error) {
```

Comment:
I'm not keen on four return arguments, I wonder if we should consider wrapping it into a `Page` object or similar?